test/generator/files/track_files_test.rb

```diff
@@ -43,9 +43,9 @@ def test_tests_template
       class TestTrackFilesUseDefault
         def initialize
           @paths = FixturePaths
-          @exercise = Exercise.new(slug: 'notemplate')
+          @slug = 'no-template'
```

Comment:
Added a hyphen so we don't need to worry about whether it's an `emplate` or not.